t/admin/plugin-metadata.t

```diff
@@ -274,7 +274,7 @@ qr/\{"error_msg":"invalid configuration: property \\"ikey\\" is required"\}/
         }
     }
 --- response_body
-{"key":"/apisix/plugin_metadata/example-plugin","value":{"ikey":1,"skey":"val"}}
+{"key":"/apisix/plugin_metadata/example-plugin","value":{"id":"example-plugin","ikey":1,"skey":"val"}}
```

Comment:
why change this?

Reply:
In the admin api, we have added an id field to the plugin metadata, so the response here needs to be added.